webui/helpers.go

```diff
@@ -415,6 +415,32 @@ func sortedLocaleNames(req *http.Request, fn func(string, bool)) {
 	}
 }
 
+func displayJobType(j *client.Job) string {
```

Comment:
This is based on https://github.com/sidekiq/sidekiq/blob/73c150d0430a8394cadb5cd49218895b113613a0/lib/sidekiq/api.rb#L374-L389. It handles regular ActiveJobs and also ActionMailer wrapped jobs, for Rails 5 and 6+.

I hope the error handling is reasonable, I think I got everything. I think that if it fails to get more specific info, it will fall back to the more general data. (E.g. if it can't get the mailer class info, it should revert to the job class, or the raw job.Type if nothing else fits.)

I do love how concise the Ruby is but it is kind of nice that the Go version forces you to handle possible errors if the data isn't in the shape that you expected. The nesting in Go seems kinda wild, not sure if there was a better way to write this.